pkg/media/manifest_builder.go

```diff
@@ -242,7 +251,7 @@ func (mb *ManifestBuilder) enhanceManifestWithMetadata(mani obj, metadata *strea
 			// Note: In the manifest, we store this in "deleteAfter" field but with timestamp value instead of duration
 			deleteAfterTimestamp := aqtime.FromMillis(expiresAtSeconds * 1000).String()
 
-			mani["assertions"].([]obj)[1]["data"].(obj)["distributionPolicy"] = obj{
+			mani["assertions"].([]obj)[1]["data"].(obj)["streamplace:distributionPolicy"] = obj{
```

Comment:
Now that we've made the decision to have the embedded Streamplace atproto records be the source of truth for how it gets distributed, I'm wondering if there's any point in having this field at all. If you can find any interoperable spec for defining distribution preferences I'd be happy to include that but if it's Streamplace-specific maybe now there's no point. I looked through the IPTC stuff without finding anything that looked like a "delete after" field (which surprised me!)